### PR TITLE
update unique test

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_mailchimp/issues/32

**This PR will result in the following new package version:** v0.5.1

Minor non breaking as we are just updating the unique test for the stg_mailchimp__members model.

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:** 

See above. This PR is in combo with the transforms [PR #40](https://github.com/fivetran/dbt_mailchimp/pull/40)

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [x] dbt run –vars (if applicable)

Need to be using `mailchimp_using_automations: true` which is the default

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

We lack internal testing data to validate this, but we know this follows the unique key combo set in the [ERD](https://docs.google.com/presentation/d/1i8JjWRgP4bDcL-TYv5flABglA_aOBXxA_OF-j1hsDcM/edit#slide=id.g127be7a8eb4_1_0) for the Mailchimp members source table.

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [ n/a] README updates have been applied (if applicable)
    
- [n/a] DECISIONLOG updates have been updated (if applicable)
- [x] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [not yet] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

**Comment by fivetran-reneeli** [comment]
- regen docs after approved

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# dbt_mailchimp_source v0.5.1
+
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:2]: Be sure to include this in a section. I would maybe classify this as a feature update as this isn't really a bug we are fixing. Just updating the test to be more accurate.

File: models/stg_mailchimp.yml [hunk 2]
```diff
@@ -475,12 +475,14 @@ models:
 
   - name: stg_mailchimp__members
     description: Each record represents a member in Mailchimp.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - member_id
+            - list_id
     columns:
       - name: member_id
```
Comment (fivetran-joemarkiewicz) [inline models/stg_mailchimp.yml:484]: Shouldn't we still be testing on not_null?

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-reneeli thanks for applying these updates! This looks good but there are a few changes to apply before approving. See my comments below.

Additionally, with the test update you will need to regenerate the docs.

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-reneeli) [reply CHANGELOG.md:2]: Added under Feature!

File: models/stg_mailchimp.yml (hunk 2 above)
Comment (fivetran-reneeli) [reply models/stg_mailchimp.yml:484]: Ah true, added

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Be sure to regen docs and then this is good to go!

**Pull request merged**